### PR TITLE
manifest: Update to PR 568

### PR DESCRIPTION
**Pull request by frkv**
-Adds nrf_cc3xx_platform/nrf_cc3xx_mbedcrypto v0.9.12

ref: NCSDK-11786

Signed-off-by: Frank Audun Kvamtrø <frank.kvamtro@nordicsemi.no>

### CODE REVIEW

**Comment by frkv** [comment]
Needs addition in known issues before this can go in

**Comment by magnev** [comment]
Full CI run is started

File: doc/nrf/known_issues.rst [hunk 1]
```diff
@@ -113,6 +113,15 @@ IRIS-2676: Missing support for FOTA on nRF Cloud
 Other issues
 ============
 
+.. rst-class:: v1-7-0 v1-6-0 v1-5-0
+
+NCSDK-11684: Failure loading KMU registers on nRF9160 devices
+  Certain builds will experience problems loading HUK to KMU due to a bug in nrf_cc3xx_platform library prior to
```
Comment (b-gent) [inline doc/nrf/known_issues.rst:119]: 'One sentence - one line' rule, please do not break sentences over multiple lines, and always start a new sentence from a new line.

The description looks good.

**Comment by b-gent** [comment]
Small changes requested

**Review by tejlmand — APPROVED** [review]

**Review by magnev — APPROVED** [review]

**Review by b-gent — APPROVED** [review]

**Pull request merged**